astronomer/providers/apache/hive/triggers/hive_partition.py

```diff
@@ -0,0 +1,86 @@
+import asyncio
+from typing import Any, AsyncIterator, Dict, Tuple
+
+from airflow.triggers.base import BaseTrigger, TriggerEvent
+
+from astronomer.providers.apache.hive.hooks.hive import HiveCliHookAsync
+
+
+class HivePartitionTrigger(BaseTrigger):
+    """
+    A trigger that fires and it looks for a partition in the given table
+    in the database or wait for the partition.
+
+    :param table: the table where the partition is present.
+    :param partition: The partition clause to wait for.
+    :param schema: database which needs to be connected in hive.
+    :param metastore_conn_id: connection string to connect to hive.
+    :param polling_period_seconds: polling period in seconds to check for the partition.
+    """
+
+    def __init__(
+        self,
+        table: str,
+        partition: str,
+        polling_period_seconds: float,
```

Comment:
Can we keep it `poll_interval` or `poke_interval` like we have in other operator/sensor

Reply:
used poke_interval variable itself